app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -364,6 +364,7 @@ open class BrowserActivity : DuckDuckGoActivity() {
             if (intent.getBooleanExtra(ShortcutBuilder.SHORTCUT_EXTRA_ARG, false)) {
                 Timber.d("Shortcut opened with url $sharedText")
                 lifecycleScope.launch { viewModel.onOpenShortcut(sharedText) }
+                return
```

Comment:
Why don't we remove this return and all subsequent ones. I think the very last statement should rather be the else branch. The code would be much cleaner